usaspending_api/search/tests/unit/test_elasticsearch_helpers.py

```diff
@@ -109,15 +109,15 @@ def test_get_download_ids(monkeypatch, transaction_type_data, elasticsearch_tran
 
 
 def test_es_sanitize():
```

Comment:
Duplicated test? I know it is pre-existing but this diff highlights the duplication with usaspending_api/search/tests/unit/test_es_sanitize.py

Reply:
Hm yeah, that is a duplicate. I think it got missed when we moved where the sanitization functions were being kept